### PR TITLE
Creating map

### PR DESCRIPTION
**Pull request by geramirez**
image

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -29,6 +31,8 @@ services:
     - 9229:9229
   client:
     image: mueller-splash-pad:client
+    stdin_open: true
```
Comment (kiranjthomas) [inline docker-compose.yaml:34]: Neat

File: client/src/App.js [hunk 2]
```diff
@@ -9,18 +9,19 @@ import {
...
+  { path: '/bartholomew', title: 'Bartholomew Park Splashpad', parkKey: 'bartholomew', lat: 30.305177, lng: -97.697469 },
+  { path: '/chestnut', title: 'Chestnut', parkKey: 'chestnut', lat: 30.277067251010195, lng: -97.71699243984315 },
+  { path: '/eastwoods', title: 'Eastwoods', parkKey: 'eastwoods', lat: 30.290943, lng: -97.731715 },
+  { path: '/liz-carpenter', title: 'Liz Carpenter Park', parkKey: 'liz-carpenter', lat: 30.26211031582992, lng: -97.75441430399374 },
+  { path: '/lott', title: 'Lott', parkKey: 'lott', lat: 30.271041206071896, lng: -97.7297671152558 },
+  { path: '/metz', title: 'Metz', parkKey: 'metz', lat: 30.252258120681006, lng: -97.71810034083994 },
+  { path: '/mueller-branch-park', title: 'Mary Elizabeth Branch Park', parkKey: 'mueller-branch-park', lat: 30.300190, lng: -97.702812 },
+  { path: '/ricky-guerrero', title: 'Ricky Guerrero Park', parkKey: 'ricky-guerrero', lat: 30.247392178325605, lng: -97.76433234845857 },
+  { path: '/rosewood', title: 'Rosewood Park', parkKey: 'rosewood', lat: 30.27109617907063, lng: -97.71390894361079 },
+  { path: '/pease', title: 'Pease Park', parkKey: 'pease', lat: 30.281754288606752, lng: -97.7521610530982 },
```
Comment (kiranjthomas) [inline client/src/App.js]: Should we get these from the server?

**Review by kiranjthomas — APPROVED** [review]
Lgtm!

File: client/src/App.js (hunk 2 above)
Comment (geramirez) [reply client/src/App.js]: yeah, I'll probably clean this up. I wanted to test the map first.

**Comment by geramirez** [comment]
I'm going to try this out in staging.

**Pull request merged**